db/Dockerfile

```
@@ -1,5 +1,8 @@
 FROM postgres:alpine
 
+COPY pg_hba.conf /data/
+COPY postgresql.conf /data/
```

Nit: could this be? (I _think_ you can use the `*` with docker images, but I'm not certain)
```suggestion
COPY *.conf /data/
```

---

Ah, yeah hold over from when I was first setting it up. Suggestion added to my next commit.